docs/stats/index.rst

```diff
@@ -77,8 +80,42 @@ with the Li \& Ma formula:
    >>> significance_on_off(n_on=18, n_off=97, alpha=1. / 10, method='lima')
    2.2421704424844875
 
+Confidence Intervals
+--------------------
+
+Assume you measured 6 counts in a Poissonian counting experiment with an
+expected background :math:`b = 3`. Here's how you compute the 90% upper limit
+on the signal strength :math:`\\mu`:
```

Comment:
The function fc_excess_limits_poisson basically combines:
fc_construct_acceptance_intervals_pdfs
fc_get_limits
fc_fix_limits
fc_find_limit

I think it's a bad idea to combine them into a function, because a) that function will be slow and b) we will have to choose a mu and x binning for the user. It would be much better to make this a class, so LowerLimit and UpperLimit aren't constantly recomputed when the function is called.

Things can always be improved.